src/index-web.html

```diff
@@ -4,6 +4,14 @@
 <head>
   <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1, user-scalable=no">
   <meta name="robots" content="noindex" />
```

Comment:
Could you add in that garbled text enforcer. Not sure what the flag is.

Reply:
done